include/RAJA/pattern/launch/launch_core.hpp

```diff
@@ -316,8 +280,7 @@ void launch(LaunchParams const& launch_params,
   using Res = typename resources::get_resource<
       typename LAUNCH_POLICY::host_policy_t>::type;
 
-  launch_t::exec(Res::get_default(), launch_params, kernel_name, p_body,
-                 reducers);
+  launch_t::exec(Res::get_default(), launch_params, nullptr, p_body, reducers);
```

Comment:
Should the kernel_name argument be removed instead of just passing nullptr now?

Reply:
sure, good call.